core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -3501,6 +3501,14 @@ public String toString() {
           .setScope(Scope.MASTER)
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .build();
+  public static final PropertyKey MASTER_METADATA_SYNC_USE_CLIENT_OPTION =
+      booleanBuilder(Name.MASTER_METADATA_SYNC_USE_CLIENT_OPTION)
+          .setDefaultValue(true)
```

Comment:
should we default it to false?

Reply:
I kinda prefer we default to true because it's more natural that files created from metadata sync respect TTL from parent directory or input params.

Reply:
Also there's gonna be quite some time before the next release so maybe there are abundant chances we validate this code in our tests and in people's deployments. I kinda prefer that community users can test this out and report if there's an issue.

Reply:
To clarify, I hope metadata sync by default creates files WITH TTL.